src/client/pages/ToeristischeVerhuur/ToeristischeVerhuur.tsx

```diff
@@ -147,7 +145,8 @@ export default function ToeristischeVerhuur() {
         );
       case 'vergunningen':
         return (
-          !(plannedVerhuur.length && cancelledVerhuur.length) || !verhuur.length
+          !(!!plannedVerhuur.length && !!cancelledVerhuur.length) ||
```

Comment:
Dit is een beetje too much. Maak er dan: `(!plannedVerhuur.length && !cancelledVerhuur.length)` van